src/brsTypes/components/RoSGNode.ts

```diff
@@ -213,6 +230,129 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
             return this.get(new BrsString(lKey));
         },
     });
+
+    /* Return the current number of children in the subject node list of children.
+    This is always a non-negative number. */
+    private getchildcount = new Callable("getchildcount", {
+        signature: {
+            args: [],
+            returns: ValueKind.Int32,
+        },
+        impl: (interpreter: Interpreter) => {
+            return new Int32(this.children.length);
+        },
+    });
+
+    /* Adds a child node to the end of the subject node list of children so that it is
+    traversed last (of those children) during render. */
+    private appendchild = new Callable("appendchild", {
+        signature: {
+            args: [new StdlibArgument("child", ValueKind.Dynamic)],
+            returns: ValueKind.Boolean,
+        },
+        impl: (interpreter: Interpreter, child: BrsType) => {
+            for (let childNode of this.children) {
```

Comment:
We may be able to reduce this down to `if (this.children.includes(child)) { return BrsBoolean.True }`.  Would you mind switching to that if you get time?

Reply:
oof what is this, built-in language features???